docs: update README (#323)
stjudecloud/workflows: README.md

* `workflows/` - Directory containing all end-to-end bioinformatics workflows.
* `tools/` - All tools we have wrapped as individual WDL tasks.
* `data_structures/` - WDL `struct` definitions and tasks or workflows related to their construction, parsing, or validation.
* `docker/` - Dockerfiles used in our workflows. All docker images are published to the [GitHub Container Registry](https://github.com/orgs/stjudecloud/packages?repo_name=workflows) as a part of our CI and are versioned.
* `scripts/` - This directory is passed to the docker build step as a context, and contains scripts that can be copied to our docker containers.

> * `test/fixtures/` - Home to all of our test fixtures. We use [Sprocket](https://sprocket.bio)'s `test` command for validating our code. Note that this directory is quite large by Git standards and we intend to prune it or otherwise reduce it's size in the future.

Reviewer: ```suggestion
* `test/fixtures/` - Home to all of our test fixtures. We use [Sprocket](https://sprocket.bio)'s `test` command for validating our code. Note that this directory is quite large by Git standards, and we intend to prune it or otherwise reduce its size in the future.
```

The workflows and tasks in this repository should require minimal set-up and configuration before you're ready to run. You don't even need to clone the repo! The bare minimum requirements are a locally installed WDL runner and an internet connection.

> The exact steps for installation, configuration, and execution are going to depend on you environment and preferred engine. There are a variety of WDL engines you could use; we ensure our WDL code executes with [Sprocket](https://sprocket.bio).

Reviewer: ```suggestion
The exact steps for installation, configuration, and execution depend on your environment and preferred engine. There are a variety of WDL engines you could use; we ensure our WDL code executes with [Sprocket](https://sprocket.bio).
```
